brage-import/src/main/java/no/sikt/nva/brage/migration/BrageImportConfig.java

```diff
@@ -0,0 +1,8 @@
+package no.sikt.nva.brage.migration;
+
+import nva.commons.core.JacocoGenerated;
+
+@JacocoGenerated
```

Comment:
The class is unused, which likely means it doesn't need to be in the PR. Generally, I'm against annotating classes like this since I generally forget to remove them when I do have coverage for them, which is a bit misleading